site/gatsby-site/src/contexts/RollbarContext.js

```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import React, { useEffect, useState, createContext, useContext } from 'react';
+import Rollbar from 'rollbar';
+import { RollbarErrorBoundary } from './RollbarErrorBoundary';
+const config = require('../../config');
+
+export const Context = createContext();
+Context.displayName = 'Rollbar';
+
+export function useRollbar() {
+  const context = useContext(Context);
+
+  const { rollbarInstance } = context;
+
+  return rollbarInstance;
+}
+
+export function RollbarProvider({ children }) {
+  const [rollbarInstance, setRollbarInstance] = useState(null);
+
+  useEffect(() => {
+    let environment = 'other';
+
+    if (window.location.hostname == 'incidentdatabase.ai') {
+      environment = 'production';
+    } else if (window.location.hostname == 'staging-aiid.netlify.app') {
+      environment = 'staging';
+    } else if (window.location.hostname == 'localhost') {
+      environment = 'localhost';
+    }
+
+    const rollbarConfig = {
+      accessToken: config.rollbar.token,
```

Comment:
What happens if the token isn't set?

Reply:
Nothing happened, the Rollbar component ignores the logging process.